src/containerapp/HISTORY.rst

```diff
@@ -12,6 +12,7 @@ upcoming
 * Add command group 'az containerapp connected-env dapr-component', support list/show/set/remove connectedEnvironments daprComponents
 * Add command group 'az containerapp connected-env storage', support list/show/set/remove connectedEnvironments storage
 * 'az containerapp env': --infrastructure-resource-group, supports custom rg name for byovnet env creations in WP enabled envs
+* 'az containerapp job create': update default properties
```

Comment:
Let me check with you if it is a breaking change? If so, it is better to add [Breaking Change] to the changlog so that customers can immediately aware of it.
```suggestion
* [Breaking Change] 'az containerapp job create': update default properties
```

Reply:
marked as breaking change as it could alter some existing customer behavior